app/views/posts/_form.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= form_for @post do |f| %>
+  <div class="form-group">
+    <%= f.label :title %><br>
+    <%= f.text_field :title, class: 'form-control' %>
+  </div>
+  <div class="form-group">
+    <%= f.label :content %><br>
```

Comment:
in the db we are using title and body, and here we are using content, who come it is running ??

Reply:
Added missing file

Reply:
ok